docs/io/fits/usage/unfamiliar.rst

```diff
@@ -136,7 +137,7 @@ type specification (i.e. the value of the TFORM keyword) uses an extra letter
 
     rPt(max)
 
-where r is 0, 1, or absent, t is one of the letter code for regular table data
+where r is the column length (it can be absent), t is one of the letter code for regular table data
```

Comment:
Thanks for the elucidating it. I got pretty confused when I saw the relation between TDIM and TFORM keywords in the FITS Standard and I thought that in VLA case r was the number of elements within a column. In the light of that, I made only minor changes guided by our discussion.